Add speed bumps to bug filing (#13086)
PowerDNS/pdns: .github/ISSUE_TEMPLATE/bug_report.md

<!-- Hi! Thanks for filing an issue. It will be read with care by human beings. Can we ask you to please fill out this template and not simply demand new features or send in complaints? Thanks! -->
<!-- Also please search the existing issues (both open and closed) to see if your report might be duplicate -->

> - [ ] This is not a support question, I have read [about opensource](https://www.powerdns.com/opensource.html) and will send support questions to the mailinglist or ask them on IRC.

Reviewer: ```suggestion
- [ ] This is not a support question, I have read [about opensource](https://www.powerdns.com/opensource.html) and will send support questions to the IRC, [Github Discussions](https://github.com/PowerDNS/pdns/pull/13086/files) or the mailing list.
```

> - [ ] I have read the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/) and am and will comply with it.

Author: Yeah, I tripped on some English...
```suggestion
- [ ] I have read the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/) and am complying with it now and will continue to comply with it.
```

> - [ ] I have read the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/) and am and will comply with it.

Reviewer: I prefer a simple "have read and understood"

> - [ ] I have read the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/) and am and will comply with it.

Reviewer: ```suggestion
- [ ] I have read and understood the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/)
```
